kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -233,6 +233,13 @@
       settingsHaveChanged() {
         return !isEqual(this.settings, this.settingsCopy);
       },
+      isPinSet() {
+        if (this.settings['extra_fields']['pin_code']) {
```

Comment:
On initial load, I get the following error in the console: 

```
TypeError: Cannot read properties of undefined (reading 'pin_code')
    at VueComponent.isPinSet (index.vue:237:1)
```

I think it would be good to do a check here, such as:

``` 
if (this.settings && this.settings['extra_fields'] && this.settings['extra_fields']['pin_code']) 
``` 

This makes sure each required piece of information is defined and present before it is used, and will help prevent undefined errors which can cause problems while loading the page.

Reply:
Thanks, Marcella!